lib/stitches/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Stitches
-  VERSION = "4.0.0-RC1"
+  VERSION = '4.0.0'
```

Comment:
Hi, sorry again to jump in.  While I don't think this change is a breaking one, if you want to bump the major version for marketing or communication reasons, can you remove this class while you are at it:

https://github.com/stitchfix/stitches/blob/master/lib/stitches/whitelisting_middleware.rb

I will try to make a quick PR that does this, but removing that class *is* a breaking change that I was planning to include whenever we had other breaking changes and, well, I obviously didn't get the chance :)  So now is a good time to do that.

Reply:
#94